rllib/env/single_agent_episode.py

```diff
@@ -16,7 +20,6 @@ def __init__(
         actions: List[ActType] = None,
         rewards: List[SupportsFloat] = None,
         infos: List[Dict] = None,
-        states=None,
```

Comment:
State outputs are no longer needed as a separate field. They are treated just like any other extra model outputs (e.g. as a (possibly nested) dict under the STATE_OUT key).